File: Java/Array/SumOfBoundaryElementOfMatrix/src/SumOfBoundary.java
public class SumOfBoundary {
    private int[][] matrix;

    // Constructor to initialize the matrix
    public SumOfBoundary(int[][] matrix) {
        this.matrix = matrix;
    }

    // Method to calculate the sum of boundary elements
    public int calculateSumOfBoundary() {
        int numRows = matrix.length;
        int numColumns = matrix[0].length;
        int sumOfBoundary = 0;

        for (int i = 0; i < numRows; i++) {
            for (int j = 0; j < numColumns; j++) {
                if (i == 0 || j == 0 || i == numRows - 1 || j == numColumns - 1) {
                    sumOfBoundary += matrix[i][j];
                }
            }
        }

        return sumOfBoundary;
    }
}

File: Python/Array/SumOfBoundaryElementsOfMatrix/SumOfBoundaryElements/SumOfBoundary.py
class SumOfBoundary:
    def __init__(self, matrix):
        self.matrix=matrix
    
    def calculateSumOfBoundary(self):
        numRows = len(self.matrix)
        numColumns = len(self.matrix[0])
        sum = 0

        for i in range(numRows):
            for j in range(numColumns):
                if i==0 or j==0 or i==numRows-1 or j==numColumns:
                    sum+=self.matrix[i][j]

        return sum
